From 4d6939143a251b93022980a472f2bcd9c42b8573 Mon Sep 17 00:00:00 2001
From: Marian Buschsieweke <marian.buschsieweke@posteo.net>
Date: Sat, 11 May 2024 18:38:51 +0200
Subject: [PATCH 3/4] RIOT integration: Select features based on modules

Using module `printf_float` adds support for printing floats, using
module `printf_long_long` adds support for printing `long long` or
`unsigned long long`.
---
 printf.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/printf.c b/printf.c
index d560d2b..6923fee 100644
--- a/printf.c
+++ b/printf.c
@@ -60,8 +60,8 @@
 #endif
 
 // support for the floating point type (%f)
-// default: activated
-#ifndef PRINTF_DISABLE_SUPPORT_FLOAT
+// default: deactivated
+#ifdef MODULE_PRINTF_FLOAT
 #define PRINTF_SUPPORT_FLOAT
 #endif
 
@@ -84,8 +84,8 @@
 #endif
 
 // support for the long long types (%llu or %p)
-// default: activated
-#ifndef PRINTF_DISABLE_SUPPORT_LONG_LONG
+// default: deactivated
+#ifdef MODULE_PRINTF_LONG_LONG
 #define PRINTF_SUPPORT_LONG_LONG
 #endif
 
-- 
2.49.0

